app/src/main/java/com/example/taskyy/ui/LoginScreen.kt

```diff
@@ -116,12 +119,12 @@ fun CreateEmailField(isEmailValid: Boolean, onEvent: (LoginEvent) -> Unit, email
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun CreatePasswordField(password: String, onEvent: (LoginEvent) -> Unit, isPasswordVisible: Boolean) {
+fun CreatePasswordField(password: String, onEvent: () -> Unit, isPasswordVisible: Boolean) {
```

Comment:
Just like for the email field, you'd also just provide a `onValueChange: (String) -> Unit` lambda here